src/map/status.c

```diff
@@ -3806,13 +3806,15 @@ void status_calc_bl_main(struct block_list *bl, /*enum scb_flag*/int flag)
 	}
 
 	if(flag&SCB_CRI && bst->cri) {
-		if (st->luk == bst->luk)
+		if (st->luk == bst->luk) {
```

Comment:
if there's conditions inside each others it's prefers to use { } even if one lined, it's easier to spot it this way.
Dastgir code style is the right here :)